src/allmydata/test/test_node.py

```diff
@@ -110,9 +111,8 @@ def test_tahoe_cfg_utf8(self):
         f.write(u"nickname = \u2621\n".encode('utf-8'))
         f.close()
 
-        n = TestNode(basedir)
-        n.setServiceParent(self.parent)
-        self.failUnlessEqual(n.get_config("node", "nickname").decode('utf-8'),
+        config = read_config(basedir, "")
+        self.failUnlessEqual(config.get_config("node", "nickname").decode('utf-8'),
```

Comment:
Do you think it's worth updating any of these deprecated assertion aliases to canonical form?